### PR TITLE
IActionResult for Spy and Enum as string

### PR DESCRIPTION
**Pull request by chiestroud**
- Updated Startup.cs to display enum items as string.
- Updated Spy.cs to add constructor to generate random ID instead of always 00000.
- Added IActionResults to Spy items I created so the error messages shows if user input wrong/non-existent ids.

### CODE REVIEW

**Review by lindseysatterfield — APPROVED** [review]
👍🏻

File: SpyDuh/Controllers/SpyController.cs [hunk 1]
```diff
@@ -19,32 +19,69 @@ public SpyController()
...
+            }
         }
 
         // Delete sklls from a spy
         [HttpDelete("{id}/skills")]
-        public object RemoveSkill(Guid id, Skills skill)
+        public IActionResult RemoveSkill(Guid id, Skills skill)
         {
-            return _repo.RemoveSkillById(id, skill);
+            if (_repo.AddSkillById(id, skill) is null)
```
Comment (jrobinson0529) [inline SpyDuh/Controllers/SpyController.cs:73]: This line isn't performing as expected. When the skill doesn't exist it still skips over this and returns the spies and not the badrequest as intended

**Review by jrobinson0529 — CHANGES_REQUESTED** [review]
Need to fix unintended result of RemoveSkill function running to remove a skill that doesn't exist

File: SpyDuh/Controllers/SpyController.cs (hunk 1 above)
Comment (chiestroud) [reply SpyDuh/Controllers/SpyController.cs]: I added a separate method for remove to check if a spy has a certain skill before removing. Thank you for pointing this out!

**Review by jrobinson0529 — APPROVED** [review]

**Pull request merged**